linera-storage-service/src/client.rs

```diff
@@ -491,6 +500,10 @@ impl AdminKeyValueStore for StorageServiceStoreInternal {
         })
     }
 
+    fn open_shared(&self, root_key: &[u8]) -> Result<Self::Store, Self::Error> {
+        self.open_exclusive(root_key)
+    }
```

Comment:
That looks like a bug.
If not, then a rename of functions is needed.

Reply:
What do you mean? There is just no difference in the implementation

Reply:
swapped